.mobconf

```diff
@@ -1,3 +1,9 @@
 [image]
 url = mobilecoin/builder-install
-tag = v0.0.18
+tag = v0.0.19
```

Comment:
IIUC this `image` section isn't used

```suggestion
```

Reply:
Left here for external scripts like in the release-scripts repo.  Will clean up in a later pr after we update or retire those scripts.